dashboard/db/migrate/20150128022616_create_ops_tables.rb

```diff
@@ -9,7 +9,7 @@ def change
       t.timestamps
     end
 
-    create_table :cohorts do |t|
+    create_table :cohorts do |t|  # rubocop:disable Style/SymbolProc
```

Comment:
And though I'm happy to rewrite it as you suggest, I feel that (at least for me) the suggestion hinders readability (especially as it breaks the pattern in the file).